cumulusci.yml

```diff
@@ -10,18 +10,61 @@ project:
       version: "1.54"
   source_format: sfdx
 
+orgs:
+  scratch:
+    demo:
+      config_file: orgs/demo.json
+      days: 30
+
 tasks:
   deploy_dev_config:
     description: Deploys the post install configuration for an unmanaged DE org
     class_path: cumulusci.tasks.salesforce.Deploy
     options:
       path: unpackaged/config/dev
 
+  dx_install_mopsy:
```

Comment:
What is mopsy?

Reply:
@mkolodner the more I think about this, the more I wonder if Mopsy should be taken out as a dependency in the package and a recommended install for users.  It would be risky to have a package we don't control as a dependency within our package.  Could cause issues for updates of ours should the package ever change.